bson/raw_bson.py

```diff
@@ -13,6 +13,59 @@
 # limitations under the License.
 
 """Tools for representing raw BSON documents.
+
+Inserting and Retrieving RawBSONDocuments
+=========================================
+
+Example: Moving a document between different databases/collections
+
+.. testsetup::
+  from pymongo import MongoClient
+  client = MongoClient("localhost", 27017, document_class=RawBSONDocument)
```

Comment:
Let's remove `localhost` and `27017` as these are the defaults used by PyMongo anyway.

Reply:
didn't know that these were defaults, removing~